File: JavaCompiler/src/app/core/models/project/project.ts
//import { GestionadorPaquete } from "src/resources/utilidades/proyecto/GestionadorPaquete";
import { Package } from "./package";

export class Project{

    public name:string;
    public principalPackage:Package;
    

    constructor(name:string){
        this.name = name;
        this.principalPackage = new Package(this.name,this.name);
    }

}

File: JavaCompilerServer/src/main/java/com/hluther/javacompiler/modelos/proyecto/Project.java
package com.hluther.javacompiler.modelos.proyecto;

public class Project {


    private String name;
    private Package principalPackage;

    public Project() {
        principalPackage = new Package();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Package getPrincipalPackage() {
        return principalPackage;
    }

    public void setPrincipalPackage(Package principalPackage) {
        this.principalPackage = principalPackage;
    }
}
